features/snippets.feature

```diff
@@ -59,77 +59,67 @@ Feature: Snippets generation and addition
 
       --- FeatureContext has missing steps. Define them with these snippets:
 
-          /**
-           * @Given /^I have magically created (\d+)\$$/
-           */
+          #[Given('/^I have magically created (\d+)\$$/')]
           public function iHaveMagicallyCreated($arg1): void
           {
               throw new PendingException();
           }
 
-          /**
-           * @When /^I have chosen '([^']*)' in coffee machine$/
-           */
+          #[When('/^I have chosen \'([^\']*)\' in coffee machine$/')]
           public function iHaveChosenInCoffeeMachine($arg1): void
           {
               throw new PendingException();
           }
 
-          /**
-           * @Then /^I should have '([^']*)'$/
-           */
+          #[Then('/^I should have \'([^\']*)\'$/')]
           public function iShouldHave($arg1): void
           {
               throw new PendingException();
           }
 
-          /**
-           * @Then /^I should get a '([^']*)':$/
-           */
+          #[Then('/^I should get a \'([^\']*)\':$/')]
           public function iShouldGetA($arg1, PyStringNode $string): void
           {
               throw new PendingException();
           }
 
-          /**
-           * @Then /^I should get a simple string:$/
-           */
+          #[Then('/^I should get a simple string:$/')]
           public function iShouldGetASimpleString(PyStringNode $string): void
           {
               throw new PendingException();
           }
 
-          /**
-           * @When /^I have chosen "([^"]*)" in coffee machine$/
-           */
+          #[When('/^I have chosen "([^"]*)" in coffee machine$/')]
           public function iHaveChosenInCoffeeMachine2($arg1): void
           {
               throw new PendingException();
           }
 
-          /**
-           * @When /^do something undefined with \\(\d+)$/
-           */
+          #[When('/^do something undefined with \\\\(\d+)$/')]
           public function doSomethingUndefinedWith($arg1): void
           {
               throw new PendingException();
           }
 
-          /**
-           * @Then /^I should have "([^"]*)"$/
-           */
+          #[Then('/^I should have "([^"]*)"$/')]
           public function iShouldHave2($arg1): void
           {
               throw new PendingException();
           }
 
-          /**
-           * @Then /^I should get a "([^"]*)":$/
-           */
+          #[Then('/^I should get a "([^"]*)":$/')]
           public function iShouldGetA2($arg1, PyStringNode $string): void
           {
               throw new PendingException();
           }
+
+      --- Don't forget these 5 use statements:
+
+          use Behat\Behat\Tester\Exception\PendingException;
+          use Behat\Step\Given;
+          use Behat\Step\When;
+          use Behat\Step\Then;
+          use Behat\Gherkin\Node\PyStringNode;
```

Comment:
I only test the output for use statements in the snippets feature. Let me know if you want to test the output in other features (I think it's not really necessary, but I may be wrong).

Reply:
I agree, we don't need it everywhere. The only other place I guess might be the i18n feature to prove your translation is coming through OK?

It's really helpful to have the full list with the PendingException, PyStringNode etc as well - thank you.

Reply:
I've also included the output test in the i18n feature file 😄